api/CSSStyleSheet.json

```diff
@@ -47,6 +47,54 @@
           "deprecated": false
         }
       },
+      "CSSStyleSheet": {
+        "__compat": {
+          "description": "<code>CSSStyleSheet()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": "66"
+            },
+            "chrome_android": {
+              "version_added": "66"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": null
```

Comment:
Did the test not work in IE? I'm pretty sure this should be false.

Reply:
I've tested https://mdn-bcd-collector.appspot.com/tests/api/CSSStyleSheet in IE11 and given the way it fails ("Object doesn't support this action") I'll indeed change this to false.

Reply:
```suggestion
              "version_added": false
```